hive-metastore/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -304,30 +305,16 @@ protected void doCommit(TableMetadata base, TableMetadata metadata) {
       throw new CommitFailedException(e);
 
     } finally {
-      cleanupMetadataAndUnlock(commitStatus, newMetadataLocation, lock);
+      HiveOperationsBase.cleanupMetadataAndUnlock(io(), commitStatus, newMetadataLocation, lock);
     }
 
     LOG.info(
         "Committed to table {} with the new metadata location {}", fullName, newMetadataLocation);
   }
 
-  @VisibleForTesting
-  Table loadHmsTable() throws TException, InterruptedException {
-    try {
-      return metaClients.run(client -> client.getTable(database, tableName));
-    } catch (NoSuchObjectException nte) {
-      LOG.trace("Table not found {}", fullName, nte);
-      return null;
-    }
-  }
+  public void setHmsParameters(
```

Comment:
It has been split into two parts , one is specific to Table like `snapshot,stats, etc.` Other pat is common parameters as ` schema, location, uuid. `

Reply:
please revert this to how it was before